modules/integration/tests-ui/src/test/java/org/wso2/carbon/am/integration/ui/VisibilityTestCase.java

```diff
@@ -0,0 +1,223 @@
+package org.wso2.carbon.am.integration.ui;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.wso2.carbon.am.integration.ui.util.APIMTestConstants;
+import org.wso2.carbon.automation.core.BrowserManager;
+import org.wso2.carbon.automation.core.ProductConstant;
+import org.wso2.carbon.automation.api.selenium.login.LoginPage;
+import org.wso2.carbon.automation.api.selenium.tenant.TenantHomePage;
+import org.wso2.carbon.automation.api.selenium.tenant.TenantListpage;
+
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+
+/**
+ * Created by nisala on 2/23/15.
+ */
+public class VisibilityTestCase extends AMIntegrationUiTestBase {
```

Comment:
This parent class has been renamed to APIMIntegrationUiTestBase. Please update your PR accordingly.